test/datatype.test.js

```diff
@@ -9,22 +9,27 @@
 /* global getSchema:false */
 var should = require('./init.js');
 
-var db, Model;
+var db = getSchema(), Model;
+var isCassandraConnector = db.connector.name === 'cassandra';
 
 describe('datatypes', function() {
   before(function(done) {
     db = getSchema();
     var Nested = db.define('Nested', {});
-
-    Model = db.define('Model', {
+    var modelTableSchema = {
       str: String,
       date: Date,
       num: Number,
       bool: Boolean,
       list: {type: [String]},
       arr: Array,
       nested: Nested,
-    });
+    };
+    if (isCassandraConnector) {
+      modelTableSchema.list = {type: String};
```

Comment:
This was a bug and fixed in another PR to connector-cassandra.  The juggler test has been amended as well.  Nice catch.